content/applications/productivity/whatsapp.rst

```diff
@@ -7,6 +7,15 @@ calls, and share content. Businesses can use `WhatsApp Business
 <https://developers.facebook.com/products/whatsapp/>`_ to communicate with their customers by text,
 send documents and provide support.
 
+.. warning::
+   WhatsApp app is an Odoo Enterprise only application that does not work in Odoo Community edition.
+   To sign up for Odoo Enterprise edition, click here: `Odoo Free Trial
+   <https://www.odoo.com/trial>`_.
+
+   .. seealso::
```

Comment:
not a fan of embedding admonitions inside other ones, can you reset the indentation here so the **seealso** occupies it's own space on the doc?